Uplift/Models/Class.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Class.swift
+//  Uplift
+//
+//  Created by Caitlyn Jin on 4/20/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+/// Model representing a class.
+import Foundation
+import UpliftAPI
+
+struct Class: Hashable {
```

Comment:
Hmm maybe we should consider renaming this to "FitnessClass" or something similar so that we don’t keep having to escape the keyword `class`